rtl/include/eclhelper_base.hpp

```diff
@@ -708,6 +707,30 @@ class CThorProjectArg : public CThorArg, implements IHThorProjectArg
     virtual bool canFilter()                                { return false; }
 };
 
+class CThorQuantileArg : public CThorArg, implements IHThorQuantileArg
+{
+    virtual void Link() const { RtlCInterface::Link(); }
+    virtual bool Release() const { return RtlCInterface::Release(); }
+    virtual void onCreate(ICodeContext * _ctx, IHThorArg *, MemoryBuffer * in) { ctx = _ctx; }
+
+    virtual IInterface * selectInterface(ActivityInterfaceEnum which)
+    {
+        switch (which)
+        {
+        case TAIarg:
+        case TAIquantilearg_1:
+            return static_cast<IHThorQuantileArg *>(this);
+        }
+        return NULL;
+    }
+
+    virtual unsigned getFlags() { return 0; }
+    virtual unsigned __int64 getNumDivisions() { return 2; }
```

Comment:
If num divisions is 2 that means it will calculate the median.  (If calculate centiles, then it calculates 99 split points.)